test-case/check-playback.sh

```diff
@@ -102,4 +102,6 @@ do
     done
 done
 
+logger_disabled || func_lib_log_post_process
```

Comment:
I think this is too early to do log post process. at this time, mtrace-reader is still running. better to add this to exit handler, see case-lib/hijack.sh.